tests/infra_test.go

```diff
@@ -688,19 +691,12 @@ var _ = Describe("[Serial]Infrastructure", func() {
 					} else {
 						resp.Body.Close()
 					}
-					errors <- err
+					errorsChan <- err
 				}(ix)
 			}
 
-			errorCount := 0
-			for ix := 0; ix < concurrency; ix++ {
-				err := <-errors
-				if err != nil {
-					errorCount += 1
-				}
-			}
-
-			fmt.Fprintf(GinkgoWriter, "client: total errors #%d\n", errorCount) // troubleshooting helper
+			errorCount, err := waitHttpResponse(errorsChan, concurrency)
+			Expect(err).To(BeNil())
```

Comment:
perhaps using this is more readable?
```suggestion
			Expect(err).ToNot(HaveOccurred())
```